jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/topicDraft.js

```diff
@@ -0,0 +1,534 @@
+                    return true;
+                }
+            }
+
+            if (data['pollItemsValue']) {
+                var pollItems = data['pollItemsValue'].split("\n");
+
+                if (pollItems.length >= MAX_ITEMS_NUMBER) {
+                    this._showError('pollItemsValue', POLL_ITEMS_VALUE_ERROR_MESSAGE);
+                    success = false;
+                } else {
+                    this._hideError('pollItemsValue');
+                }
+
+                if (success) {
+                    pollItems.forEach(function(item) {
+                        if (item && (item.length >= MAX_POLL_ITEM_LENGTH)) {
+                            success = false;
+                        }
+                    });
+
+                    if (!success) {
+                        this._showError('pollItemsValue', POLL_ITEM_LENGTH_ERROR);
+                    } else {
+                        this._hideError('pollItemsValue');
+                    }
+                }
+            } else {
+                this._hideError('pollItemsValue');
+            }
+
+            return success;
+        },
+
+        _showError: function(element, message) {
+            if (this._errors[element]) {
+                this._errors[element].error.text(message);
+                this._errors[element].group.addClass('error');
+            }
+        },
+
+        _hideError: function(element) {
+            if (this._errors[element]) {
+                this._errors[element].error.text('');
+                this._errors[element].group.removeClass('error');
+            }
+        },
+
+        _getData: function () {
+            var draft = {
+                title: this._title.val() ? this._title.val() : undefined,
+                content: this._content.val() ? this._content.val() : undefined
+            };
+
+            if ((this._pollTitle && this._pollTitle.val()) ||
+                (this._pollItemsValue && this._pollItemsValue.val())) {
+                draft['pollTitle'] = this._pollTitle.val();
+                draft['pollItemsValue'] = this._pollItemsValue.val();
+            }
+
+            return draft;
+        },
+
+        _addEventListener: function (event, callback) {
+            this._title.on(event, callback);
+            this._content.on(event, callback);
+
+            if (this._pollTitle) {
+                this._pollTitle.on(event, callback);
+            }
+
+            if (this._pollItemsValue) {
+                this._pollItemsValue.on(event, callback);
+            }
+        },
+
+        _onBlur: function () {
+            var self = this;
+
+            this._timer.stop();
+            if (this._wasChanged && this._validate()) {
+                this.save().then(function () {
+                    self._wasChanged = false;
+                });
+            }
+        },
+
+        _onKeyUp: function (event) {
+            var self = this;
+
+            this._wasChanged = true;
```

Comment:
This means that the rules about when the draft is getting saved are changed. Previously we were checking if the actual data were changed and only if it did we were sending the request to the server. I'm more or less okay with your approach, but we should notify the team about this decision in the comments to the JIRA issue.